setup.py

```diff
@@ -164,14 +164,16 @@ def write_version(filename=os.path.join(*['airflow',
 redis = ['redis>=2.10.5']
 kubernetes = ['kubernetes>=3.0.0',
               'cryptography>=2.0.0']
+mongo = ['pymongo>=3.6.0']
```

Comment:
There should be an entry down in `extras_require` as well so someone can choose to install without specifying `all_dbs`.